src/sentry/tasks/delete_seer_grouping_records_by_hash.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any
+
+from sentry import options
+from sentry.seer.similarity.grouping_records import delete_grouping_records_by_hash
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task
+
+
+@instrumented_task(
+    name="sentry.tasks.delete_seer_grouping_records_by_hash",
+    queue="delete_seer_grouping_records_by_hash",
+    max_retries=0,
+    silo_mode=SiloMode.REGION,
+)
+def delete_seer_grouping_records_by_hash(
+    project_id: int,
+    hashes: list[str],
+    last_deleted_index: int = 0,
+    *args: Any,
+    **kwargs: Any,
+) -> None:
+    """
+    Task to delete seer grouping records by hash list.
+    Calls the seer delete by hash endpoint with batches of hashes of size `BATCH_SIZE`.
+    """
+    batch_size = options.get("embeddings-grouping.seer.delete-record-batch-size")
+    len_hashes = len(hashes)
+    end_index = min(last_deleted_index + batch_size, len_hashes)
+    delete_grouping_records_by_hash(project_id, hashes[last_deleted_index:end_index])
+    if end_index < len_hashes:
```

Comment:
any reason to call the task again instead of just calling the `delete_grouping_records_by_hash` function repetitively?

Reply:
it's based on the logic we used for the backfill - I didn't want this task to timeout, so I'm respawning the task instead

Reply:
what is our current timeout set to on the celery task?

Reply:
haha whoops just realized I didn't even add a timeout, but I was thinking  of using the same timeouts from the backfill `soft_time_limit=60 * 15`, `time_limit=60 * (15 + 5)`

Reply:
got it. i'd consider thinking about increasing the batch size to 100, and optionally doing some calls in a loop, so that way we have to spawn less celery tasks, as each one should run pretty quick (less than a second)

Reply:
nice, I didn't realize they were that quick - thanks!